src/commcare_cloud/commcare_cloud.py

```diff
@@ -88,7 +88,8 @@ def run_on_control_instead(args, sys_argv):
     branch = getattr(args, 'branch', 'master')
     cmd_parts = [
         executable, args.env_name, 'ssh', 'control', '-t',
-        'source ~/init-ansible && git checkout master && control/update_code.sh && git checkout {branch} && git reset --hard origin/{branch} && source ~/init-ansible && {cchq} {cchq_args}'
+        'source ~/init-ansible && git fetch && git checkout {branch} '
```

Comment:
It might be worthwhile to add the `--prune` flag on fetch

it looks like this no longer checks for changes on the current branch, but that seems fine since it's on control and it's hard reseting?

Reply:
Yeah if the branch is master, than this will update the master branch, etc. That's basically the optimization, skipping updating an irrelevant branch.

What's the scenario you're thinking `--prune` would help with? (I might just be ignorant of the pattern.)

Reply:
oh, basically just having too many git objects around. it's probably not a huge concern, but afaik old merged branches would just hang around forever as is.

Reply:
okay I'll add that. thanks!